src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AssemblyAlignmentParser.java

```diff
@@ -114,20 +113,25 @@
     @VisibleForTesting
     static Iterable<AlignmentRegion> breakGappedAlignment(final AlignmentRegion oneRegion, final int sensitivity) {
 
-        final List<CigarElement> cigarElements = oneRegion.forwardStrandCigar.getCigarElements();
+        final List<CigarElement> cigarElements = new ArrayList<>();
+        final boolean cigarHasInsertionAsTrailingOperator = parseCigarAndConvertTrailingInsertionToSoftClip(oneRegion.cigarAlong5to3DirectionOfContig, cigarElements);
```

Comment:
And because of the assumption that's now enforced in the argument checking step on gap operators not neighboring the clipping operators, "trailing" actually means not the first or the last operator.